riscv-plic.adoc

```diff
@@ -17,102 +18,291 @@ https://creativecommons.org/licenses/by/4.0/.
 This document contains the RISC-V platform-level interrupt controller (PLIC)
 specification, which defines an interrupt controller specifically designed to
 work in the context of RISC-V systems.  The PLIC multiplexes various device
-interrupts onto the external interrupt lines of various hart contexts, with
-hardware support for interrupt priorities.
+interrupts onto the external interrupt lines of various Hart contexts, with
+hardware support for interrupt priorities. +
+This specification defines the general PLIC architecture and operation parameters.
+The PLIC which claimed as PLIC-Compliant standard PLIC should follow the
+implementations mentioned in below sections.
```

Comment:
It sounds better with "sections below" instead :-)

Reply:
issue addressed